HPXMLtoOpenStudio/resources/hpxml_schematron/EPvalidator.xml

```diff
@@ -828,6 +833,7 @@
       <sch:report role='WARN' test='number(h:ExposedPerimeter) &gt; 2*number(h:Area)'>Slab exposed perimeter is more than twice the slab area, this may indicate an input error.</sch:report>
       <sch:report role='WARN' test='number(h:Thickness) &lt; 1 and number(h:Thickness) &gt; 0'>Thickness is less than 1 inch; this may indicate incorrect units.</sch:report>
       <sch:report role='WARN' test='number(h:Thickness) &gt; 12'>Thickness is greater than 12 inches; this may indicate incorrect units.</sch:report>
+      <sch:report role='WARN' test='count(h:ExteriorHorizontalInsulation) >0 and count(h:PerimeterInsulation)=0'> There is ExteriorHorizontalInsulation but no PerimeterInsulation, this may indicate an input error.</sch:report>
```

Comment:
We should add a test in `test_validation.rb` to demonstrate that this warning works correctly. See the `test_schema_schematron_warning_messages` tests, where you configure an HPXML to generate the warning and then check that it matches the expected warning message.

Once you add this test, you'll see that this line is not right. At a minimum, you should be using `number()` instead of `count()`. Also, you can't use `>` inside of the `<...>` xml elements, because it will think that's the end of the xml tag, rather you have to use `&gt;`. I'm kind of confused that it didn't cause an error...

Reply:
Remaining...

Reply:
Addressed.